src/Symfony/Bundle/TwigBundle/DependencyInjection/TwigExtension.php

```diff
@@ -92,6 +92,13 @@ public function load(array $configs, ContainerBuilder $container)
             $dir = dirname($reflection->getFileName()).'/Resources/views';
             if (is_dir($dir)) {
                 $this->addTwigPath($twigFilesystemLoaderDefinition, $dir, $bundle);
+
+                $classMock = new $class();
```

Comment:
What about this; as bundles are initialized before the container is ([ref](https://github.com/symfony/symfony/blob/973f29551255cac22b87e9e82ff93a280c9959c6/src/Symfony/Component/HttpKernel/Kernel.php#L114)) we can easily inject this kind of information. Currently `kernel.bundles` giving you only the FQCN is not flexible. We sometimes need the path, parent etc. So im thinking a special VO that replaces the FQCN is worth it... for now as additional parameter perhaps, but eventually to deprecate `kernel.bundles` being a list of FQCN's...

I would like this a lot!